integr8ly/keycloak-operator-rhmi
# ensure client secrets are recreated

## JIRA ID
INTLY-6495

## Additional Information
The client secret is not recreated when deleted because the check only covers the client itself: if the client exists (but not the client secret), the operator will attempt to update the non-existing secret and fail with a nil pointer exception.

## Verification Steps
1. Deploy the redhat-rhmi-rhsso namespace in a RHMI installation
2. Wait for the operator to recreate it
3. The operator should not crash and the client secrets should be created in the rhsso namespace (after keycloak itself becomes ready).

## Checklist:
